modules/core/04-channel/keeper/handshake_test.go

```diff
@@ -562,132 +561,6 @@ func (suite *KeeperTestSuite) TestChanCloseInit() {
 	}
 }
 
-// TestChanCloseConfirm tests the confirming closing channel ends by calling ChanCloseConfirm
-// on chainB. Both chains will use message passing to setup OPEN channels. ChanCloseInit is
-// bypassed on chainA by setting the channel state in the ChannelKeeper.
-func (suite *KeeperTestSuite) TestChanCloseConfirm() {
```

Comment:
Why was this test removed. Only upgrade case should be removed

Reply:
Huh, good point. Will check that!

Reply:
Added back